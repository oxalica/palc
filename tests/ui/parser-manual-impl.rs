use palc::Parser;

struct Cli {}

impl Parser for Cli {}

fn main() {}
