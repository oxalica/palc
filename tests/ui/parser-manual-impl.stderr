error[E0277]: the trait bound `Cli: ParserInternal` is not satisfied
 --> tests/ui/parser-manual-impl.rs:5:17
  |
5 | impl Parser for Cli {}
  |                 ^^^ the trait `ParserInternal` is not implemented for `Cli`
  |
note: required by a bound in `Parser`
 --> src/lib.rs
  |
  | pub trait Parser: ParserInternal + Sized + 'static {
  |                   ^^^^^^^^^^^^^^ required by this bound in `Parser`
